src/test/java/org/folio/fqm/migration/strategies/TestTemplate.java

```diff
@@ -25,14 +27,31 @@ public abstract class TestTemplate {
   /** [Description, Source, Expected] */
   public abstract List<Arguments> getExpectedTransformations();
 
+  public List<Arguments> getExpectedTransformationsWithAdditionalAssertions() {
+    return getExpectedTransformations()
+      .stream()
+      .map(args -> {
+        if (args.get().length == 3) {
+          return Arguments.of(args.get()[0], args.get()[1], args.get()[2], null);
+        } else {
+          return args;
+        }
+      })
+      .toList();
+  }
+
```

Comment:
added an optional parameter to call extra assertions after migrating (mostly for mockito)